### ISSUE TITLE
Lesson 1: Recognizing cats and dogs

### ISSUE THREAD

**Issue by bkowshik** [bkowshik/100-days-of-fastai#3]
### General resources
- [ ] [How to Ask for Help](http://wiki.fast.ai/index.php/How_to_ask_for_Help)
- [ ] [Where can I put my Jupyter Notebook?](https://github.com/reshamas/fastai_deeplearn_part1/blob/master/tips_faq_beginners.md#q2--where-can-i-put-my-jupter-notebook)
- [ ] [Platforms for using fastai / bash setup](https://github.com/reshamas/fastai_deeplearn_part1/blob/master/README.md#platforms-for-using-fastai-gpu-required) and [Paperspace setup](https://github.com/reshamas/fastai_deeplearn_part1/blob/master/tools/paperspace.md)

### Lesson resources
- [ ] [Lesson video](https://youtu.be/IPBSB1HLNLo)
- [ ] Clone the fastai repo: `https://github.com/fastai/fastai.git`
- [ ] [Introductions thread](http://forums.fast.ai/t/how-has-your-journey-been-so-far-learners/6480/57): Bit about yourself and what you're hoping to achieve
- [ ] [Lesson notes](https://medium.com/@hiromi_suenaga/deep-learning-2-part-1-lesson-1-602f73869197) from hiromi
- [ ] [Lesson notes](http://forums.fast.ai/t/deeplearning-lec1notes/7089) from timlee
- [ ] [Pre-release videos of fast.ai Intro to Machine Learning for Coders](http://forums.fast.ai/t/another-treat-early-access-to-intro-to-machine-learning-videos/6826?source_topic_id=9398)

### Blogs mentioned in the lesson
- [ ] [Image Kernels](http://setosa.io/ev/image-kernels/)
- [ ] [Convolution > Non Linear > Convolution animation](http://neuralnetworksanddeeplearning.com/chap4.html)
- [ ] [Linear algebra cheatsheet for deep learning](https://medium.com/towards-data-science/linear-algebra-cheat-sheet-for-deep-learning-cd67aba4526c)
- [ ] [CNNs from different viewpoints](https://medium.com/impactai/cnns-from-different-viewpoints-fab7f52d159c)
- [ ] [Transformations means](http://forums.fast.ai/t/wiki-lesson-2/9399/21?u=ecdrid)

### Other links from the lesson
- [ ] [Learning rate paper](https://arxiv.org/abs/1506.01186)
- [ ] [Resnet model](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py)
- [ ] [Kaggle](https://www.kaggle.com/surveys/2017)

### Homework
- [ ] Setup environment on crestle/aws/paperspace
- [ ] Get comfortable with Jupyter and all other tools
- [ ] Run week 1 code and pay with code to understand it
- [ ] Try different learning rates, epochs while running code
- [ ] Feel free to explore week 2 notebook

### Video Resources:
- [ ] [Friendly intro to CNNs](https://www.youtube.com/watch?v=2-Ol7ZB0MmU)
- [ ] [CS 231N: CNN Lecture](https://www.youtube.com/watch?v=GYGYnspV230)
- [ ] [The Universal Approximation Theorem for neural networks](https://www.youtube.com/watch?v=Ijqkc7OLenI)

### Podcasts
- [ ] [CNN Mini episode](https://dataskeptic.com/blog/episodes/2017/convolutional-neural-networks)
- [ ] [Talking Machines](http://www.thetalkingmachines.com/)
- [ ] [This week in Machine learning (TWIML)](https://twimlai.com/)
- [ ] [The AI Podcast](https://soundcloud.com/theaipodcast)

### Other links
- [ ] [Build a Taylor Swift detector](https://towardsdatascience.com/build-a-taylor-swift-detector-with-the-tensorflow-object-detection-api-ml-engine-and-swift-82707f5b4a56)
- [ ] [Starting deep learning hands-on: image classification on CIFAR-10](https://blog.deepsense.ai/deep-learning-hands-on-image-classification/)
- [ ] [Use Google images to download lots of training data](https://github.com/boxabirds/fastai-helpers)
